x-pack/docs/en/security/authentication/user-profile.asciidoc

```diff
@@ -0,0 +1,86 @@
+[role="xpack"]
+[[user-profile]]
+=== User Profile
+
+beta::[]
+
+User Profile provides a way to uniquely and stably refer to a user.
+Unique means each user has an identifier that is guaranteed to be unique within
+a deployment. Stable means the identifier remains generally unchanged in the
+lifetime of a deployment.
```

Comment:
I think this starts on the wrong footing. Are users not unique enough that we need profiles for it?

Reply:
I agree it's a bad openning statement because I didn't try to imply that users are not unique enough. In fact, it is quite the opposite, users (username + realm) are too unique. But we probably don't want these details in the beginning either. So I think the missing keyword here is "persistent" because users can be empheral but profiles are always persisted.